automatic/python3/python3.nuspec

```diff
@@ -31,6 +31,9 @@ For example: `--params '"/InstallDir:C:\tools\python2"'`
     <licenseUrl>https://www.python.org/download/releases/3.4.0/license</licenseUrl>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/python3</packageSourceUrl>
     <projectSourceUrl>https://www.python.org/downloads/source</projectSourceUrl>
+    <dependencies>
+      <dependency id="chocolatey" version="0.10.8" />     
```

Comment:
why a dependency on chocolatey, instead of chocolatey-core.extension?

Reply:
Because of Get-PackageParameters

Reply:
which is available in the extension, which would be better to have as a dependency instead of dictating which choco versions users must use IMO.